hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoCreateHandlerTest.java

```diff
@@ -256,6 +279,39 @@ void preHandleWorksWhenInitialBalanceIsZero() throws PreCheckException {
         assertEquals(key, context.payerKey());
     }
 
+    @Test
+    @DisplayName("preHandle succeeds when has non zero evm alias")
+    void preHandleWorksWhenHasEvmAlias() throws PreCheckException {
+        final byte[] evmAddress = CommonUtils.unhex("6aeb3773ea468a814d954e6dec795bfee7d76e26");
+        txn = new CryptoCreateBuilder()
+                .withAlias(Bytes.wrap(evmAddress))
+                .withStakedAccountId(3)
+                .build();
+        final var context = new FakePreHandleContext(readableStore, txn);
+        subject.pureChecks(txn);
+        subject.preHandle(context);
+
+        assertEquals(txn, context.body());
+        basicMetaAssertions(context, 1);
+        assertEquals(key, context.payerKey());
+    }
+
+    @Test
+    @DisplayName("preHandle fails when invalid alias key")
+    void preHandleWorksWhenHasAlias() throws PreCheckException {
+        final Bytes SENDER_ALIAS =
+                Bytes.fromHex("3a21030edcc130e13fb5102e7c883535af8c2b0a5a617231f77fd127ce5f3b9a620591");
+        txn = new CryptoCreateBuilder()
+                .withAlias(SENDER_ALIAS)
+                .withStakedAccountId(3)
+                .build();
+        final var context = new FakePreHandleContext(readableStore, txn);
+        subject.pureChecks(txn);
```

Comment:
What's the purpose of calling `pureChecks(...)` here?

Reply:
Good catch 👍  
Removed